bigtable-client-core-parent/bigtable-v1-protos/README.md

```diff
@@ -0,0 +1,76 @@
+# Bigtable Protos
```

Comment:
I think that bigtable-legacy-protos is better.  We have users who directly use the bigtable-protos maven artifacts, so I'd rather not change the name.